grammars/dream maker.cson

```diff
@@ -25,7 +25,7 @@ patterns: [
 	}
 	{
 		match: '''(?x)
-		(var)[\\/ ]
+		(?:^|[\\s\\t\\(])(var)[\\/ ]
```

Comment:
Wouldn't it make more sense/be sufficient to use `\w`?

Reply:
some schizos can possibly use non-English as variable name(although it's not a thing). `\s` `\t` `\(` `being first` would be all cases where `var/` is used.

Reply:
I meant `\b`, sorry.

Reply:
you're right. \b is cleaner.